popups.css

```diff
@@ -13,6 +13,12 @@
 .lsp_popup li {
     font-family: system;
 }
+.lsp_popup h1,
+.lsp_popup h2,
+.lsp_popup h3,
+.lsp_popup h4 {
```

Comment:
You could combine those rules to avoid duplication.

Reply:
I'm not very good at CSS. I think I combined the selectors in my latest commit?

Reply:
Perfect.